File: submissions/sm_008_chaitanya/week_03/day_4/session_1/mirrorImage.js
function mirrorImage(input) {
    var cap = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    var small = "abcdefghijklmnopqrstuvwxyz"
    var x, y;
    var result = " "
    for (i = 0; i < input.length; i++) {
        for (j = 0; j < cap.length; j++) {
            if (input[i] == cap[j]) {
                result += cap[25 - j]
            }
            else if (input[i] == small[j]) {
                result += small[25 - j]
            }
        }
    }
    return result
}
console.log(mirrorImage("AyzB"))

File: submissions/sm_028_sagar/week_13/day_4/session_1/mirror_image.py
a = 'AyzB'

cap = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
small = 'abcdefghijklmnopqrstuvwxyz'
string = ''

for i in range(0,len(a)):
    for j in range(0,len(cap)):
        if(cap[j] == a[i]):
            string += cap[-(j+1)]
    for k in range(0,len(small)):
        if(small[k] == a[i]):
            string += small[-(k+1)]
print(string)